skeleton-tests/test_addon_content_type.py

```diff
@@ -142,6 +142,52 @@ def test_addon_content_type(tmpdir, capsys, config):
     )
     assert result == 0
 
+    # generate subtemplate content_type with parent container:
+    template = """[variables]
+dexterity_type_name=parent
+dexterity_type_base_class=Container
+dexterity_type_create_class=True
+dexterity_type_global_allow=True
+dexterity_type_filter_content_types=True
+subtemplate_warning=False
+"""
+    generate_answers_ini(wd, template)
+
+    config.template = 'content_type'
+    result = subprocess.call(
+        [
+            'mrbob',
+            'bobtemplates.plone:' + config.template,
+            '--config', 'answers.ini',
+            '--non-interactive',
+        ],
+        cwd=wd,
+    )
+    assert result == 0
+
+    # generate subtemplate content_type with child container:
+    template = """[variables]
+dexterity_type_name=parent
```

Comment:
parent again?
You have to create the children here and make the correct settings for it, as you would test it manually.